server/config/seed.js

```diff
@@ -23,15 +23,18 @@ var Project = require('../api/project/project.model');
 var Post = require('../api/post/post.model');
 var Smallgroup = require('../api/smallgroup/smallgroup.model');
 var ClassYear = require('../api/classyear/classyear.model')
+var Attendance = require('../api/attendance/attendance.model')
 
 
 // Seed the database with the original sample data
-var seed = function() {
+var seed = function(cb) {
```

Comment:
this function should return a promise and should not have a callback- this is the new javascript paradigm